### PR TITLE
Address Validation: Related to issue #71

### PR DESCRIPTION
**Pull request by KarahanS**
issue #71  by @WazeHell

### CODE REVIEW

**Comment by KarahanS** [comment]
Aim is to validate the address by checking whether or not it belongs to one of the cities from the corresponding region.

**Comment by WazeHell** [comment]
there's typo in "Hatay'"

File: applications/tweets/helpers.py [hunk 1]
```diff
@@ -11,6 +11,23 @@
 
 logger = logging.getLogger(__name__)
 
+def create_address(processed_address=None,full_address=None,tweet=None):
+    city = processed_address.get("city")
+    allowed_cities = ["Gaziantep", "Malatya", "Batman", "Bingöl", "Elazığ", "Kilis", "Diyarbakır", "Mardin", "Siirt", "Şırnak", "Van", "Muş", "Bitlis", "Hakkari", "Adana", "Osmaniye","Hatay"]
```
Comment (okankaraduman) [inline applications/tweets/helpers.py:16]: Bunu yapmak yerine aşağıdaki regexi kullanarak uppercase ve typoları da yakalayan bir kurgu yapalım
```suggestion
(?i)((gaz[ıiİI]antep)|(malatya)|(batman)|(b[ıiIİ]ng[oöOÖ]l)|(elaz[Iİıi][gğ])|(kilis)|(diyarbak[ıiIİ]r)|(mardin)|(siirt)|([SsŞş][ıiIİ]rnak)|(van)|(mu[sşSŞ])|(bitlis)|(hakkari)|(adana)|(osmaniye)|(hatay)|(kahramanmara[sşSŞ])|(mara[SŞsş])|(antep))

```

**Review by okankaraduman — CHANGES_REQUESTED** [review]
comment'e göre değiştirmeliyiz.

**Comment by KarahanS** [comment]
* Added regex for cities.
* Resolved merge conflict.
You can test the regex using the code below:
```python
import re
allowed_cities = '(?i)((gaz[ıiİI]*antep)|(malatya)|(batman)|(b[ıiIİ]*ng[oöOÖ]*l)|(elaz[Iİıi]*[gğ])|(kilis)|(diyarbak[ıiIİ]*r)|(mardin)|(siirt)|([SsŞş]*[ıiIİ]*rnak)|(van)|(mu[sşSŞ]*)|(bitlis)|(hakkari)|(adana)|(osmaniye)|(hatay)|(kahramanmara[sşSŞ]*)|(mara[SŞsş]*)|(antep))'
allowed_cities_re = re.compile(allowed_cities)
city  = "biingOl"
if re.search(allowed_cities_re, city):
    print("match!")
else:
   print("No match!")
```

**Comment by KarahanS** [comment]
Invalid as the structure of [helpers](https://github.com/acikkaynak/deprem-yardim-backend/blob/main/applications/tweets/helpers.py) has been changed.

**Pull request closed without merging**